edn/src/symbols.rs

```diff
@@ -69,10 +69,11 @@ pub struct NamespacedKeyword {
 }
 
 impl PlainSymbol {
-    pub fn new(name: &str) -> Self {
```

Comment:
Consider extending this to the `Namespaced` variants too.  I've filed https://github.com/mozilla/mentat/issues/199 for this.